kurtosis_package/dockerfiles/contract_deployer.Dockerfile

```diff
@@ -1,46 +1,31 @@
-FROM debian:bookworm-slim AS foundry
+FROM debian:bookworm-slim
```

Comment:
We can use `ghcr.io/foundry-rs/foundry` here directly

Reply:
https://github.com/Layr-Labs/avs-devnet/pull/221/commits/270b87405a7dd4b17119762fd33816e950127669